### ISSUE TITLE
feature: improve load times for large notes

### ISSUE THREAD

**Issue by AnthonyMcDougal** [philips/supernote-obsidian-plugin#19]
Large multi-page notes take a while (~30 sec? depending on length) to load, and this repeats every time the file is selected for the open pane. 

1. Is there a way to decrease this time?
2. Is there a way to not repeat this every time the file is selected, or is exporting the image the only way?

**Comment by philips** [comment]
Yeah unfortunately parsing the images is slow. I am sure there is a way to speed that up but I don’t know when I would have time to dig into that. 

If someone were want to take a performance pass at the library the file is in here https://github.com/philips/supernote-typescript/blob/main/src/conversion.ts

**Comment by philips** [comment]
Added generation of a cpuprofile to one of the tests to start to hunt this down: https://github.com/philips/supernote-typescript/commit/57f19b7a2a5722cadf06dbb817ab53111ad4b0e2

**Comment by philips** [comment]
I just merged #44 which reduces parsing of a 10 page file from 7s to about 1.5s for me. Please try out the beta release for 2.4.0 via BRAT. I would appreciate your testing.

**Comment by philips** [comment]
The latest beta has a ton of rendering speed ups. At this point I think it is the best I can do.
